resources/views/customer/customer/mail.blade.php

```diff
@@ -0,0 +1,19 @@
+@component('mail::message')
+Hello, {{$customer['name']}}.
+
+This is your statement.
+
+
+@component('mail::table')
+| Date     | Due Date     | Amount |   Payment | Balance
+|:--------|:-------------|:-------|:---------|:-------|
+@foreach($receipts as $receipt)
+| {{$receipt['date']}} | {{$receipt['due_date']}} | {{$receipt['grand_total']}} | {{$receipt['total_amount_received']}} | {{$receipt['grand_total']-$receipt['total_amount_received']}} |
+@endforeach
+@endcomponent
+
+Please pay the above before or on {{$receipt['due_date']}}.
```

Comment:
I think the `due date` varies by receipt. In that case, consider adding a separate column for that.